sync/ui.md

```diff
@@ -20,16 +26,41 @@ There are 4 main screens in the initial Admin UI:
 
 ### Data Schema
 
-x
+*TODO*: why a resolver per property (see screenshot)? Should it not be one resolver per type?
+*TODO*: is the left side editor interactive (should i be able to add/delete nodes in there)?
```

Comment:
ping @david-martin

Reply:
Yes, it is interactive.
The right side is a visual representation of the schema